content/blog/2017/06/2017-06-26-share-jenkins-world-keynote-stage.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Come Share the Jenkins World Keynote Stage with Me!"
+tags:
+- event
+- jenkinsworld
+author: Kohsuke Kawaguchi
+---
+
+link:https://www.cloudbees.com/jenkinsworld[Jenkins World] is approaching fast,
+and we are all busy preparing for the event.
```

Comment:
Since this isn't a guest post, I think the "we" should be re-worded to be less ambiguous